examples/ftd_api_scripts/ips_access_rule.py

```diff
@@ -70,7 +73,23 @@ def main():
     result = post_access_rule(host, port, access_token, access_rule)
     if not result:
         print('Unable to post access rule')
-        return
+        return False
+    else:
+        return True
 
 if __name__ == '__main__':
-    main()
+    import sys
+
+    if len(sys.argv) != 6:
+        print("Usage: python ftd_api_scripts/ips_access_rule.py host port user passwd policy_name")
```

Comment:
for policy_name, what should this look like from the command line?  
put an example here.

Reply:
Updated all 3 scripts that use policy_name with an example message